solidity@4.rb

```diff
@@ -0,0 +1,37 @@
+#------------------------------------------------------------------------------
+# solidity.rb
+#
+# Homebrew formula for solidity.  Homebrew (http://brew.sh/) is
+# the de-facto standard package manager for OS X, and this Ruby script
+# contains the metadata used to map command-line user settings used
+# with the 'brew' command onto build options.
+#
+# Our documentation for the solidity Homebrew setup is at:
+#
+# http://solidity.readthedocs.io/en/latest/installing-solidity.html
+#
+# (c) 2014-2017 solidity contributors.
+#------------------------------------------------------------------------------
+
+class SolidityAT4 < Formula
+  desc "The Solidity Contract-Oriented Programming Language"
+  homepage "http://solidity.readthedocs.org"
+  url "https://github.com/ethereum/solidity/releases/download/v0.4.25/solidity_0.4.25.tar.gz"
+  version "0.4.25"
+  sha256 "8172c126973eae1d9d6c7b5071910a9c7475ac64df3945f755cee66104add72d"
+
+  depends_on "cmake" => :build
+  depends_on "boost@1.60" => "c++11"
```

Comment:
Why is this needing a specific version while `0.5.x` doesn't?

Reply:
This gets a little beyond my C++ knowledge, but iirc, something changed with the boost versions used recently, and if you build this version of Solidity with the latest version of boost, you get errors. When I researched those errors, it seemed that the change happened in recent boost versions, this is the closest Homebrew version of boost available that works with compiling this version of Solidity. You can try it if you like and see what I mean, but this fixed those build errors.

Reply:
@ekpyron @chriseth do you happen to know the reason? If we release 0.4.26 for fixing the Firefox inconvenience, we may as well fix this one.

Reply:
I don't know - 1.60 sounds a little old, too...

Reply:
Well the Solidity cmake does check for the minimum supported version so this should be safe.

Reply:
> 1.60 sounds a little old, too...

Keep in mind this is compiling 0.4.x which assumed 1.54 as a minimum.

Reply:
@chriseth and @axic These are sadly the version available - https://formulae.brew.sh/formula/boost

I couldn't get a build to work with the 1.69 version, but I am not sure why, I can share logs if need be. Or we know of a specific boost version that is newer, and does work, i can make a PR to their formula?